python/lsst/pipe/tasks/dcrAssembleCoadd.py

```diff
@@ -220,15 +229,14 @@ class DcrAssembleCoaddTask(CompareWarpAssembleCoaddTask):
 
     ConfigClass = DcrAssembleCoaddConfig
     _DefaultName = "dcrAssembleCoadd"
-    makeIdFactory = _makeMakeIdFactory("CoaddId")
 
     def __init__(self, *args, **kwargs):
         super().__init__(*args, **kwargs)
         if self.config.doCalculatePsf:
-            schema = afwTable.SourceTable.makeMinimalSchema()
-            self.makeSubtask("detection", schema=schema)
-            self.makeSubtask('measurement', schema=schema)
-            self.makeSubtask("measurePsf", schema=schema)
+            self.schema = afwTable.SourceTable.makeMinimalSchema()
+            self.makeSubtask("detectSourcePsf", schema=self.schema)
+            self.makeSubtask('measurement', schema=self.schema)
```

Comment:
Can we get some consistent double vs. single quote action while you're at it, please? It's also not immediately clear to me why there is a `measurement` and a `measurePsf` subtasks as separate things. I really like how you've updated the name from `detection` --> `detectSourcePsf`. Would it be possible to do the same for `measurement` which is pretty vague?

Reply:
How is `measurePsfSources`? I would probably also change the detection task to `detectPsfSources` for consistency.

Reply:
I think that's good! It helps clarify that you're detecting and measuring the PSF sources first, and then (presumably) measuring the PSF itself later.